sdk/sdkmanifest/plan.go

```diff
@@ -0,0 +1,427 @@
+package sdkmanifest
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"os"
+
+	"go.autokitteh.dev/autokitteh/internal/kittehs"
+	"go.autokitteh.dev/autokitteh/sdk/sdklogger"
+	"go.autokitteh.dev/autokitteh/sdk/sdkservices"
+	"go.autokitteh.dev/autokitteh/sdk/sdktypes"
+)
+
+const defaultEnvName = "default"
```

Comment:
"default" is not a good default name, because we may have other defaults in the future. Maybe just "env"?

Reply:
it's a default environment name. i think it's apt.